config/archaeology/config_split.config_split.archaeology.yml

```diff
@@ -10,9 +10,11 @@ module: {  }
 theme: {  }
 blacklist:
   - config_split.config_split.archaeology
-  - 'menu_position.menu_position_rule.*'
 graylist:
-  - '*'
+  - migrate_plus.migration.hs_events_image_importer
```

Comment:
I thought we were getting rid of all site-specific config?

Reply:
I am planning to remove site specific configs after this PR. I found that we actually _can_ have site specific configs as long as its very explicit with the graylist and blacklist. Notice the removal of the `*`. This will keep the sites functioning in the same way after this code release. 

After that, i can then add an update hook to remove the site specific split and alter the configs in such a way that keeps them running as they exist now.